stdlib/InteractiveUtils/src/codeview.jl

```diff
@@ -353,7 +353,7 @@ const llvm_types =
 const llvm_cond = r"^(?:[ou]?eq|[ou]?ne|[uso][gl][te]|ord|uno)$" # true|false
 
 function print_llvm_tokens(io, tokens)
-    m = match(r"^((?:[^\s:]+:)?)(\s*)(.*)", tokens)
+    m = match(r"^((?:[^%\s:]+:|\"[^:]*\":)?)(\s*)(.*)", tokens)
```

Comment:
```suggestion
    m = match(r"^((?:[^\"\s:]*:|\"[^\"]*\":)?)(\s*)(.*)", tokens)
```
If I were to take your intention, it would be this?